packages/amplify_storage_s3/example/android/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:3.5.4'
```

Comment:
Is it ok to just make this change here, or should we try to have the same version in all these files? IMO, ok to have this patch here as it's an example app, after all. This is needed here because of something related to new version of file_picker which has sound nulls safety https://github.com/miguelpruivo/flutter_file_picker/wiki/Troubleshooting#-issue.